src/enums/filingCodes.ts

```diff
@@ -5,5 +5,6 @@ export enum FilingCodes {
   FM_CONVERSION = 'FMCONV',
   FM_CORRECTION = 'FMCORR',
   INCORPORATION_BC = 'BCINC',
-  INCORPORATION_CP = 'OTINC'
+  INCORPORATION_CP = 'OTINC',
+  SPECIAL_RESOLUTION='SPRLN'
```

Comment:
Should have space before and after equal sign.

Please run `npm run lint` locally to find these.